tests/user_test.py

```diff
@@ -3,10 +3,14 @@
 from fastapi.testclient import TestClient
 from models.user import User
 from tests.test_fixtures import create_and_teardown_tables, client
-
+from io import BytesIO
+import os, hashlib
 @pytest.fixture(scope="function", autouse=True)
 def setup_and_teardown_db() -> Generator[TestClient, None, None]:
     yield from create_and_teardown_tables(User.metadata)
+    
+def hash_email(email: str) -> str:
```

Comment:
It is not as straightforward as you think, what if user Joel has a profile pictures then changes its email address suddenly. The hash is going to be different and we won;t be able to get his picture anymore.